lib/action_controller/serialization.rb

```diff
@@ -57,5 +61,13 @@ def serialization_scope(scope)
         self._serialization_scope = scope
       end
     end
+
+    def original_url
+      request.original_url.sub(/\?.*$/, "")
```

Comment:
Or `request.original_url[/\A[^?]+/]` which has the advantage of using an anchor :)